collect_app/src/main/java/org/odk/collect/android/fragments/FileManagerFragment.java

```diff
@@ -129,8 +129,9 @@ public void onLoadFinished(@NonNull Loader<Cursor> loader, Cursor cursor) {
         deleteButton.setEnabled(areCheckedItems());
 
         if (getListView().getCount() == 0) {
-            toggleButton.setEnabled(false);
+            getView().findViewById(R.id.buttons).setVisibility(View.GONE);
```

Comment:
This was being hidden before because the empty view took up the full available vertical space which pushed the buttons out of the frame. Now we hide the buttons explictly. I'd usually test this kind of change (as I did for the commit before), but this code should be rewritten soon as part of #5343.